tests/structuredheadings/formatcombo.js

```diff
@@ -33,23 +38,50 @@
       });
     },
 
-    "if previous heading is autonumber, the new heading is autonumbered": function () {
+    "increments autonumbering levels and preset": function () {
```

Comment:
Can you take another crack at this name? What increments autonumbering? Is "preset" a term that's sufficient on its own?